tests/test_client.py

```diff
@@ -0,0 +1,108 @@
+import os
+
+from  dotenv import load_dotenv
+import pytest
+
+import fairly
+
+
+load_dotenv('./.env')
+FIGSHARE_TOKEN = os.environ['FAIRLY_FIGSHARE_TOKEN']
+DUMMY_DATASET = "./tests/dummy_dataset/"
```

Comment:
As the package will support different clients, it can be better to have unit tests for each client separately.